docs/speech-usage.rst

```diff
@@ -145,5 +145,82 @@ words to the vocabulary of the recognizer.
     transcript: Hello, this is a test
     confidence: 0.81
 
+
+Streaming Recognition
+---------------------
+
+The :meth:`~google.cloud.speech.Client.stream_recognize` method converts speech
+data to possible text alternatives on the fly.
+
+.. note::
+    Streaming recognition requests are limited to 1 minute of audio.
+
+    See: https://cloud.google.com/speech/limits#content
+
+    >>> import io
+    >>> from google.cloud import speech
+    >>> from google.cloud.speech.encoding import Encoding
+    >>> client = speech.Client()
+    >>> with io.open('./hello.wav', 'rb') as stream:
+    >>>     sample = client.sample(stream=stream, encoding=Encoding.LINEAR16,
+    ...                            sample_rate=16000)
+    >>>     stream_container = client.stream_recognize(sample)
+    >>> print(stream_container)
+    <google.cloud.speech.streaming.container.StreamingResponseContainer object at 0x10538ee10>
+    >>> print(stream_container.responses)
+    {0: <google.cloud.speech.streaming.response.StreamingSpeechResponse object at 0x10f9ac9d0>}
+    >>> print(stream_container.responses[0].results[0].alternatives[0].confidence)
```

Comment:
This is just a mouthful. Why put readers through it? Just describe the types returned and point to documentation for those types.